src/App.ts

```diff
@@ -400,10 +418,10 @@ export default class App {
     // Factory for say() utility
     const createSay = (channelId: string): SayFn => {
       const token = context.botToken !== undefined ? context.botToken : context.userToken;
-      return (message: Parameters<SayFn>[0]) => {
+      return (message: Parameters<SayFn>[0]): Promise<void | WebAPICallResult> => {
```

Comment:
I'm still thinking but it may be better to return `Promise<WebAPICallResult | CodedError>` here even after applying `onGlobalError`. Any thoughts?

Reply:
I clearly wasn't thinking straight here, I removed the global error handler as it will be called anyway if this goes uncaught. 

This means that this returns what `chat.postMessage` is which is simply `Promise<WebAPICallResult>`

I just checked on CodedError, looks cool, I would think the right thing to do would be to update `@slack/web-api` to handle that rather than bolt. That way you have a consistent api between `say` and `client.chat.postMessage` What do you think?

Reply:
🙏 for the late reply here: I agree that we can remove the global error handler here to be consistent with others such as  `app.client` calls and `respond` 👍